.gitignore

```diff
@@ -11,3 +11,7 @@
 config/certs
 config/settings/*.local.yml
 /spec/examples.txt
+
+
+.rbenv-gemsets
```

Comment:
would this go better in a global .gitignore? https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Is there a standard .gitignore we all use?